src/Uno.UI.Tests/Windows_UI_Xaml/Given_Window.cs

```diff
@@ -9,6 +9,7 @@ namespace Uno.UI.Tests.Windows_UI_Xaml;
 public class Given_Window
 {
 	[TestMethod]
+	[Ignore]
```

Comment:
Why is this needed ? If you have issues here, Martin may be able to help.

Reply:
For the purpose of this PR, these tests should indeed not be ignored.

The tests are problematic when run in parallel because they don't cleanup `Window.Current`, and it already fails in CI, but when the CI retries the failing test individually the CI gets eventually green.

So, I agree the tests have problems, but they shouldn't still be ignored and CI is expected to pass.

Note: This is something I'll take a look at with https://github.com/unoplatform/uno/pull/12901

Reply:
I asked @MartinZikmund about this and he noted that these will need to be changed for the upcoming multi-window support anyway, so ignored them in their current (kind of broken) state is okay. I've undone the `Ignore`s.